WordPress/gutenberg
# Prevent scripts from loading if behaviors are not used

## What?
When no behaviors are used, do not load the `interactivity-view.js` file.

## Why?
So that we don't wastefully load JS.

## How?
Adding a workaround very similar to the one [currently used](https://github.com/WordPress/gutenberg/blob/5ec913ff88158a13837cb88fdf3c7effb2f7aa05/packages/block-library/src/navigation/index.php#L648-L660) in the Navigation block.

## Testing Instructions
1. Open a Post or Page.
2. Insert an Image Block.
3. Open the post preview and make sure that the `interactivity-view.js` file is not loaded.
4. In the **Advanced** panel select the `Lightbox` behavior and save the post.
5. Go back to the preview and make sure that this time the file IS being loaded.
6. Try the same thing with more than one Image block.
